### DISCUSSION TITLE
help for the game source code.

### DISCUSSION THREAD

**Discussion by mmcme** [roll-w/CSL2-StationNaming#19] [Q&A] [answered]
it is a good example of modding, as i am just learnning how to develop a mod.
i have a few question while i read your code:
1.UI mod.  there is a moduleRgistry extend   to " game-ui/game/components/selected-info-panle/selected-info-sections/selected-info-sections.tsx", so ,where can i get these code?  the game code : *.tsx .

2.code part, we need to "using Game, using Game.UI, using Game.buildings.TransportStation...." how can i get these information ?

 i want to write a mod to query the count of waiting people at some stops, how can i get these informantion. which entity ,which component i should use .such these questions, i can not get any anwsers from wikis of paradox
i even can not get the stop name, if i have not read your code.
thanks.

**Comment by roll-w** [comment, marked as answer]
You may need to decompile the game dll to get more code information since the official wiki obviously won't include everything we want. Which entities and components to use depends on your needs and can be determined by reading the code.

For UI code, you should enable UI debug to obtain UI information within the game (follow the official wiki), but to get started quickly, looking at someone else's implementation might be better.
